app/_hub/kong-inc/application-registration/how-to/_index.md

```diff
@@ -36,6 +36,64 @@ curl -X PATCH http://localhost:8001/plugins/<plugin_id> \
   --data "config.auto_approve=true"
 ```
 
+### Enable proxy with consumer's credential
+
```

Comment:
Activating application registration will restrict access to the service using consumer credentials. To regain this functionality, you can enable the `enable_proxy_with_consumer_credential` flag.